### DISCUSSION TITLE
Multiple Datasets

### DISCUSSION THREAD

**Discussion by juanFernigrini** [ericgio/react-bootstrap-typeahead#781] [Ideas]
Hi @ericgio 
 im using your typeahead and its really good, yesterday i was doing a new page and i wasnt able to resolve this case.
 
If you are looking for citi from states, you must choose one state and after that you choose your city, but whit the example at the en "multiple dataset" you can type your city and look what state is yours.

https://twitter.github.io/typeahead.js/examples/  <--- Multiple Datasets
https://gist.github.com/jharding/9458797#file-multiple-datasets-js  <--- example

**Comment by ericgio** [comment]
Hi @juanFernigrini please describe in detail what you're trying to do and what isn't working. Code samples or a link to a sandbox with your work are very helpful.

**Reply by juanFernigrini** [reply]
im tryig to mix recent search whit suggestions, separated by a little title beetwenn.

**Reply by ericgio** [reply]
@juanFernigrini: please have a look at the ["Custom menu" example](https://ericgio.github.io/react-bootstrap-typeahead/#rendering). It shows how you can group your options into different sections.
